telnyx/api_resources/address.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import absolute_import, division, print_function
+
+from telnyx.api_resources.abstract import (
+    CreateableAPIResource,
+    DeletableAPIResource,
+    ListableAPIResource,
+)
+
+
+class Address(CreateableAPIResource, DeletableAPIResource, ListableAPIResource):
+    OBJECT_NAME = "address"
+
+    @classmethod
+    def class_url(cls):
+        return "/v2/addresses"
```

Comment:
Don't want to write a fully generic pluralizer for en-US? :P

Reply:
I considered it for about a minute before landing on this solution 🙃